src/FunctionApps/python/FhirServerExport/__init__.py

```diff
@@ -27,12 +27,11 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
         container = ""
 
     cred_manager = AzureFhirServerCredentialManager(fhir_url=fhir_url)
```

Comment:
From @bamader: Flagging this as something that will need to be changed once the refactor lands. It'll become `azure.AzureFhir...`. 

Response from @spencer-skylight: This has been addressed.